### DISCUSSION TITLE
Is it free to join codespaces beta?

### DISCUSSION THREAD

**Discussion by Jonathan0827** [community/community#17034] [Codespaces] [answered]
# Is it free to join Codespaces?
## In personal account

**Comment by dommilosz** [comment, marked as answer]
Yes. You only need to register in waitlist and wait.

**Reply by Jonathan0827** [reply]
# Thank you!
![Thanks](https://user-images.githubusercontent.com/74815888/169673438-8934937d-2217-45da-b077-adad376f78dc.gif)

**Reply by hidp123** [reply]
> Yes. You only need to register in waitlist and wait.

Can you please link to the waitlist, I can't find it. Thanks

**Reply by dommilosz** [reply]
I would search there https://github.com/features/codespaces but I don't see the link. I think it was there
In FAQ they write:
How do I get access to Codespaces? 
> Codespaces is rolling out progressively on August 11th, 2021 and [can be enabled in settings](https://github.co/enable-codespaces) by organization owners for Team and Enterprise Cloud plans. For users in individual plans, we’re extending the existing Codespaces beta. For those in the beta, access will remain and we’ll share updates on what’s coming in the near future.
